### PR TITLE
Add instructions on how to update the Hail TLS certs on Google Cloud

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

File: hail_batch_dev.md [hunk 1]
```diff
@@ -249,3 +249,62 @@ When users submit a batch, they specify a billing project which will be charged
...
+   hail_dir = os.getenv("HAIL")
+   x = yaml.safe_load(open(f"{hail_dir}/tls/config.yaml"))["principals"]
+   print(",".join(x["name"] for x in x))')
+
+   kubectl delete pods -l "app in ($SERVICES_TO_RESTART)"
+   ```
+
+1. [Verify in a browser](https://batch.hail.populationgenomics.org.au/) that the expiration date for the certificate has been extended.
+
+1. Don't forget to stop the [`hail-setup` VM](https://console.cloud.google.com/compute/instancesDetail/zones/australia-southeast1-b/instances/hail-setup?project=hail-295901). Please don't delete it!
```
Comment (illusional) [inline hail_batch_dev.md:310]: @lgruen, can you create a machine image for the hail-setup VM so if it does get deleted we don't have to set it up again?

Comment (lgruen) [reply hail_batch_dev.md:310]: There's actually nothing special going on there (I think). It'll be interesting to see whether @violetbrina will run into any issues, as she might not have used the VM before, whereas I've got a bunch of settings in my profile.

The main reason to use a VM in the first place is to have a much faster Docker build experience and better network bandwidth for pulling / pushing images.

**Review by violetbrina — APPROVED** [review]
I'm not that knowledgable about the gcp deployment and updating the certificates. But in my ignorance at least the instructions added here seem clear. 

I'm assuming the hail-setup vm is equivalent to the Azure bootstrap-vm for deploying hail? In which case at least the instructions would be very similar, excluding the platform specific guide for connecting into the Azure VM. 

If they are the same thing I've setup the Azure bootstrap vm multiple times so that part will be clear. I can add some instructions to this doc when I go through the process again to setup our Azure stack.

**Comment by lgruen** [comment]
@violetbrina Thanks! Do you want to try following the instructions? We'll need to update the certs before the end of the month anyway, so this would be a good opportunity to check that everything is documented correctly.

File: hail_batch_dev.md (hunk 1 above)
Comment (violetbrina) [reply hail_batch_dev.md:310]: Ok I've successfully gone through the whole process. The only (I suspect one time thing) I had to do was install the gke-gcloud-auth-plugin component on the gcloud cli to get everything to work.

My suggestion on the docs though is *how* exactly to check the expiration date for the certificate. That is unclear and was the one thing I couldn't do.

**Review by violetbrina — CHANGES_REQUESTED** [review]
Clarify the **Verify in a browser** step

File: hail_batch_dev.md (hunk 1 above)
Comment (lgruen) [reply hail_batch_dev.md:315]: Thanks! I've updated the instructions to also mention installing the gke-gcloud-auth-plugin and provided a command for the TLS certificate expiration date check.

If you want to do that in the browser, you can click the lock icon:
CleanShot 2023-05-09 at 08 20 58@2x

Interestingly, your update did *not* extend the expiration date:

```bash
> echo | openssl s_client -servername batch.hail.populationgenomics.org.au -connect batch.hail.populationgenomics.org.au:443 2>/dev/null | openssl x509 -noout -dates

notBefore=Feb 28 21:17:11 2023 GMT
notAfter=May 29 21:17:10 2023 GMT
```

Are you sure the `letsencrypt` step (6) finished successfully?

**Review by violetbrina — APPROVED** [review]

**Pull request merged**